### DISCUSSION TITLE
🎄 v0.3.2 Holiday Release!

### DISCUSSION THREAD

**Discussion by roncodes** [fleetbase/fleetbase#174] [Announcements]
🎄 **Holiday Cheers from Fleetbase!** 🌟

Ho Ho Ho! 🎅 As we celebrate this festive season, we are thrilled to unveil our latest gift to you – **Fleetbase v0.3.2**! This isn't just any update; it's a sleigh-full of major enhancements and critical patches that will supercharge your logistics experience. 🚀

**What’s New in Your Stocking:**

- 📍 **Multi-Order Import with Lat/Long Coordinates:** Your wish for seamless bulk order importing is granted! Now import multiple orders effortlessly using latitude and longitude.
- 📱 **Navigator App Settings & Instance Linking:** Link up! Our new "Instance Linking" creates a unique URL from your Fleetbase instance, opening the URL will trigger the Navigator app installed on your phone to open and update it's connection to your instance, it will also update the app branding to yours!
- 🗺️ **Choose Your GPS in Navigator App:** We’ve ditched the Mapbox dependency. Now, navigate using any GPS service you fancy.
- 💨 **Frontend Supercharged with Ember 5.4:** Feel the speed with the latest Ember 5.4, loaded with performance upgrades that will make your dashboard fly!
- 🔧 **Required Node v18 for Fleetbase Console:** Upgrade for enhanced stability and features – we've moved from v16 to v18.
- 🛣️ **Enhancements in Multiple Waypoint Order Creation:** Smoother, more efficient routing for complex deliveries.
- 🛠️ **Critical API Patches:** We’ve patched up to ensure a more robust and secure API.
- ⚓ **Helm Chart for GCP & Kubernetes:** Deploying on Google Cloud Platform or Kubernetes? Our new Helm chart makes it a cinch.
- 🖥️ **UX Improvements:** Toggle sidebar visibility for a more customized experience and enjoy an improved upload interface.
- 💵 **Support for All Currencies in Money Input:** Now, no matter where you are, we support your currency.

🌟 **Upgrade to v0.3.2 Now and Experience the Magic!** 🌟

This release is our way of saying thank you for your continued trust in Fleetbase. We're committed to making your logistics smoother, faster, and more intuitive. So, unwrap this update and let's drive into a brighter, more efficient future together!

**Happy Holidays and Happy Logistics!**

With Warm Wishes, The Fleetbase Team

**Comment by devonbreary72-ctrl** [comment]
💳🎛20028 time auzususususuaudhudhushdysyeyeyeydysyeyesyeyeyrysyeysyryywysyeyyeryyeyeyeyryeyeysyeyryeyeyyeyryeydyryeydyydyeyryeyeyyeyeyeydyeyeyyydyyeyyfyeysyrt
